composer.json

```diff
@@ -86,7 +86,7 @@
         "oomphinc/composer-installers-extender": "^1.1",
         "drupal/access_unpublished": "^1.0-alpha8",
         "drupal/adsense": "^1.0",
-        "drupal/admin_toolbar": "^2.0",
+        "drupal/admin_toolbar": "2.0",
```

Comment:
Should this maybe <2.1. If someone is using the beta and only wants to update thunder

Reply:
But I don't have strong opinions on this. You know what I think about people that are using outdated software 😶

Reply:
I was actually quite shocked, that you brought this up ;-)